lib/colors.ts

```diff
@@ -153,6 +153,10 @@ export function toPercent(value: number): string {
   return `${clean(100 * value)}%`
 }
 
+export function isHexNotation(value: string): boolean {
+  return /^#(?:[A-Fa-f0-9]{3}){1,2}$/.test(value)
```

Comment:
There is also 4-8 digits hex for alpha `#00000088`/`#0008` (equal to `rgba(0, 0, 0, 0.5)`)

Reply:
Fixed it